blog/src/pages/blog.js

```diff
@@ -72,34 +71,31 @@ class IndexPage extends React.Component {
           {this.props.data.allAsciidoc.edges.map(({ node, idx }) => {
             if (node.document.title !== "Author") {
               const formattedDate = this.formatDate(node.fields.slug);
-              const authorImageSource = (
-                "../../images/images/avatars/" +
-                node.pageAttributes.author +
-                ".jpg" ??
-                "../../images/images/avatars/" +
-                node.pageAttributes.author +
-                ".png" ??
-                "../../images/images/avatars/" +
-                node.pageAttributes.author +
-                ".jpeg" ??
-                "../../images"
-              );
+              const authorNames = node.pageAttributes.author ?? "author"
+              const authorNamesWoSpaces = authorNames.replace(/\s/g, '')
+              const authorArray = authorNamesWoSpaces.split(",")
+              authorArray.forEach(element => {
+
+                element = "../../images/images/avatars/" + element + ".jpg"
```

Comment:
To be honest this is not really acceptable, as we do not want to restrict the format of the images used.

Reply:
@Vandit1604 Would you like me to take a look later to see what else we could do when I have the time?

Reply:
You may need to revert the image format changes as a result

Reply:
@krisstern Yeah sure because I have tried everything comparisons and everything but it's just not working. It would be a great help Kris if you take a look when you have time.

Reply:
> You may need to revert the image format changes as a result

Yeah sure just a minute

Reply:
Hi @Vandit1604 You have not reverted the images to their original formats yet. Are you planning to do that?

Reply:
Nevermind, let me revert the changes

Reply:
@krisstern I have done that. see here https://github.com/Vandit1604/jenkins-docs/pull/60/commits/59f89db3583a8760fb96cb9c7fa6c9362d67b7c1

Reply:
The changes you made previously are not showing up: https://github.com/Vandit1604/jenkins-docs/tree/modify-blog-pages/blog/static/images/images/avatars